tests/integration/browser.migration.js

```diff
@@ -46,13 +46,7 @@ describe('migration', function () {
       if (!match) {
         return testUtils.Promise.resolve();
       }
-      return new testUtils.Promise(function (resolve, reject) {
-        var script = document.createElement('script');
-        script.onload = resolve;
-        script.onerror = reject;
```

Comment:
the asyncLoadScript does not have the error handler, would you mind adding it there with the promise rejection as here?

Reply:
in addition, i think asyncLoadScript will fail if there is no existing script tag due to prepending document.getElementsByTagName('script')[0], if we replace existing code we might also add the fallback behaviour there to fall back to append child to body if there is no existing script tag.

Reply:
Good catch - thanks!  I'll update.

Reply:
> we might also add the fallback behaviour there to fall back to append child to body if there is no existing script tag

I've updated to just append all scripts to `document.body`.  It seems to work 🫣